app/helpers/earthworks_blacklight_helper.rb

```diff
@@ -2,7 +2,7 @@ module EarthworksBlacklightHelper
   include Blacklight::LayoutHelperBehavior
   # Blacklight override for show main content
   def show_content_classes
```

Comment:
Can you remove this method and just override `main_content_classes` now?  https://github.com/projectblacklight/blacklight/blob/c4ce13eee7eb0819c72f35a46db4eb565e6dfaa6/app/helpers/blacklight/layout_helper_behavior.rb#L32C1-L34C8